cluster-up/cluster/kind/common.sh

```diff
@@ -303,7 +303,7 @@ function down() {
     if [ -z "$($KIND get clusters | grep ${CLUSTER_NAME})" ]; then
         return
     fi
-    $KIND delete cluster --name=${CLUSTER_NAME}
+    $KIND delete cluster -v 9 --name=${CLUSTER_NAME}
```

Comment:
btw after it fails, the pod is indeed there ?
can you please do docker ps -a after wards in case of failure ?

worth to ask kind slack community maybe

thanks for looking into it